docs/conf.py

```diff
@@ -116,20 +116,13 @@ def setup(app):
 # The theme to use for HTML and HTML Help pages.  See the documentation for
 # a list of builtin themes.
 #
-html_theme = 'sphinx_rtd_theme'
+html_theme = 'alabaster'
```

Comment:
sphinx_rtd_theme has dependencies that conflict with the current version of bibtex, so one of sphinx's default themes was used.